README.md

```diff
@@ -1 +0,0 @@
-# PuppetLogic
```

Comment:
Why would we remove the README from this repo? I think it is always good to have kind of an "intro" and links and useful stuff when you first open the repo

Reply:
I'd say we want to keep this repo as clean and light as possible. All the information can be given on the px repository. I don't see anything that should be here instead